### PR TITLE
DUT remove return process

### PR DESCRIPTION
**Pull request by clairlin53**
Add return process in HW transfer tools.

The commit introduces the return process in the HW transfer tools,
1. separate the content and test result, making it more flexible to use.
2. allowing automatic changes to the C3 status when DUTs are no longer necessary,
such as the SI or DVT1 phases.
This automation streamlines the workflow and ensures accurate status updates.

Test: https://warthogs.atlassian.net/browse/OQS-3309

### CODE REVIEW

File: Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 
-from C3.apis.base import C3API, TaipeiLocation
+from C3.apis.base import C3API, C3Location
```
Comment (hanhsuan) [inline Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py:3]: I couldn't see the `Taipeilocation` in `API/C3/apis/base.py` is changed to `C3Location`. Does it miss or some magic that I don't know?

Comment (clairlin53) [reply Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py:3]: Sorry, forgot to modify it, I have updated, thank you.

**Review by hanhsuan — APPROVED** [review]
LGTM

**Review by baconYao — APPROVED** [review]
+1

**Pull request merged**